plugins/org.yakindu.sct.doc.user/src/Part2-User-Guide/40_editing_statecharts.textile

```diff
@@ -71,7 +71,7 @@ h3(#editor-palette). Editor palette
 
 The editor palette provides you with a set of various actions and statechart editing tools. By default, the palette is located right of the canvas, but you can also drag it to the left.
 
-You can hide the palette by clicking the small triangle in the palette's title bar. Click the triangle again to make the palette reappear.
+You can hide the palette by clicking at the small triangle in the palette's title bar. Click on the triangle again to make the palette reappear.
```

Comment:
Why "clicking **at** the small triangle" and next sentence "Click **on** the triangle". Please be consistent. Either use "on" or leave the preposition out.